modules/standard/IO.chpl

```diff
@@ -5482,19 +5571,19 @@ record itemReaderInternal {
 // And now, the toplevel items.
 
 /* standard input, otherwise known as file descriptor 0 */
-const stdin:channel(false, iokind.dynamic, true);
+const stdin:_channel(false, iokind.dynamic, true);
```

Comment:
Would it make sense to turn this into a fileReader officially?  And the other two into fileWriters?